app/dashboard/static/js/app/view-socs-soc-job-kernel.2017.4.js

```diff
@@ -20,381 +23,45 @@
-                                '%s out of %s',
-                                format.number(
-                                    uniqueTotal.totals.defconfig),
-                                format.number(totalBuilds)))
-                    );
-                } else {
-                    tooltipNode.appendChild(
-                        document.createTextNode(
-                            format.number(uniqueTotal.totals.defconfig)));
-                }
-
-                html.replaceContent(
-                    document.getElementById('unique-defconfigs'), tooltipNode);
-            } else {
-                html.replaceContent(
-                    document.getElementById('unique-defconfigs'),
-                    html.nonavail());
-            }
-        } else {
-            html.replaceByClassNode('unique-values', html.nonavail());
-        }
-
-        if (Object.getOwnPropertyNames(uniqueLab).length > 0) {
-            for (lab in uniqueLab) {
-                if (uniqueLab.hasOwnProperty(lab)) {
-                    localLab = uniqueLab[lab];
-                    labStatus = localLab.status;
-
-                    if (labStatus.hasOwnProperty('fail')) {
-                        failCount = labStatus.fail;
-                    } else {
-                        failCount = 0;
-                    }
-
-                    if (labStatus.hasOwnProperty('pass')) {
-                        passCount = labStatus.pass;
-                    } else {
-                        passCount = 0;
-                    }
-
-                    if (labStatus.hasOwnProperty('unknown')) {
-                        unknownCount = labStatus.unknown;
-                    } else {
-                        unknownCount = 0;
-                    }
-
-                    totalCount = passCount + failCount + unknownCount;
-
-                    html.replaceContent(
-                        document.getElementById('boot-count-' + lab),
-                        createLabBootCount(
-                            totalCount, passCount, failCount, unknownCount)
-                    );
-
-                    html.replaceContent(
-                        document.getElementById('unique-count-' + lab),
-                        createOtherCount(localLab.totals));
-                }
-            }
-        }
-    }
-
-    function getBootDoneUnique(response) {
-        var deferred;
-
-        if (response.count > 0) {
-            deferred = request.get(
-                '/_ajax/count/build', {
-                    job: gJob,
-                    kernel: gKernel
-                });
-
-            $.when(deferred, unique.countD(response))
-                .fail(error.error, uniqueCountFail)
-                .done(uniqueCountDone);
-        } else {
-            html.replaceByClassTxt('unique-values', '?');
-        }
-    }
-
     function getBootsFailed() {
         html.replaceContent(
             document.getElementById('accordion-container'),
```

Comment:
I don't think there are any `accordion-container` elements any more in this view.  Can you locally get this function called instead of `getBootsDone` to make it show the error message?

Reply:
This method is not necessary anymore, I removed it! Now the code is using same logic from job page, as you can see here: https://github.com/alexandrasp/kernelci-frontend/blob/t18936-add-soc-sub-view-test-plan/app/dashboard/static/js/app/view-socs-soc-job-kernel.2017.4.js#L429